<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resultado</class>
 <widget class="QDialog" name="resultado">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>631</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QLabel" name="lblImagen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblWHistigrama">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtnProcesar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Procesar</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="ptxtDescripcion">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>101</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QRadioButton" name="rbtnGray">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Escala de grises</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbtnColor">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
